enterprise/server/remote_execution/runner/runner.go

```diff
@@ -1245,6 +1367,34 @@ func (p *pool) Shutdown(ctx context.Context) error {
 		if len(runnersToRemove) > 0 {
 			log.Infof("Runner pool: removing %s", runnerSlice(runnersToRemove))
 		}
+
+		for _, r := range pausedRunners {
```

Comment:
Because we're only saving state of paused runners, does that mean if all runners are executing tasks during a restart, it will take longer to boot up the runner pool?